package.json

```diff
@@ -115,12 +115,12 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^12.22.0 || >=14.15.0",
+    "node": "^14.15.0 || >=16.13.0",
     "yarn": "^1.17.0"
   },
   "engines-next": {
     "description": "Versions other than the below are deprecated and a warning will be logged",
-    "node": ">=14.15.0"
+    "node": "^14.15.0 || >=16.13.0"
   },
   "dependencies": {
```

Comment:
https://github.com/renovatebot/renovate/blob/665b3d359cf5db486550afb32d593de2096386b9/.github/workflows/build.yml#L36-L54

Do we want/need to change our build/test matrix as well?

Reply:
#12244 😉